src/services/user.service.js

```
@@ -0,0 +1,49 @@
+const { User } = require('../models/User.model');
+
+User.sync({ force: true });
+
+const getUsers = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getUser = async (id) => {
+  return User.findByPk(id);
+};
+
+const createUser = async (name) => {
+  const user = await User.create({
+    name,
+  });
+
+  return user;
+};
+
+const deleteUser = async (id) => {
+  const user = await getUser(id);
+
+  if (!user) {
+    return null;
+  }
+
+  await User.destroy({ where: { id } });
+
+  return user;
+};
+
+const updateUser = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
+
+  const updatedUser = await getUser(id);
```

`updateUser` calls `User.update` then fetches the updated user. If the id does not exist, `getUser` will return `null` and that's returned to the caller. Consider validating the presence of the user before updating and/or normalizing incoming fields (e.g. ensure `name` is a string) so the service fails fast with predictable outputs.